templates/dev.rake

```diff
@@ -1,4 +1,4 @@
-if Rails.env.development? || Rails.env.test?
+unless Rails.env.production?
```

Comment:
I.. hope not?
I see that you [started getting rid of staging](https://github.com/thoughtbot/suspenders/pull/743), but it looks like it [was](https://github.com/thoughtbot/suspenders/blob/8c39efccac0fad8e6b74d25ff3a8809dd2f7df00/templates/secrets.yml#L8) [never](https://github.com/thoughtbot/suspenders/blob/master/templates/Gemfile.erb#L49) [completed](https://github.com/thoughtbot/suspenders/blob/a930e6bf9fbc4fb11bf59fbf637349a0e3733fad/templates/postgresql_database.yml.erb#L21).



We still use staging/production environments -- and staging PR environments are tagged as staging at the ENV level as well.

While I agree that [Heroku made that recommendation](https://devcenter.heroku.com/articles/multiple-environments#managing-staging-and-production-configurations), I don't understand it. It's like taking a config option that was built for environment separation and no longer using it.